src/App.jsx

```diff
@@ -14,33 +15,85 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPrepareGoods(goods, { sortField, isReversed }) {
+  const prepareGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (sortField) {
+    case 'alphabetically':
+      prepareGoods.sort((a, b) => a.localeCompare(b));
+      break;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    case 'byLength':
+      prepareGoods.sort((a, b) => a.length - b.length);
+      break;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    prepareGoods.reverse();
+  }
+
+  return prepareGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The `sortField` state is initialized as an empty string. It might be more intuitive to initialize it with a default value like 'alphabetically' or 'byLength' to ensure the goods are sorted by default when the component first renders.